.github/workflows/docs.yml

```diff
@@ -93,6 +93,16 @@ jobs:
         if: always()
         run: if [ -e doc/errors.txt ]; then cat doc/errors.txt; fi
 
+      - name: Test Documentation Images
+        run: pytest tests/doc/tst_doc_images.py
+
+      - name: Upload Test Debug Images
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: doc-debug-images
+          path: _doc_debug_images
+
```

Comment:
Perhaps the `docs.yml` workflow should only upload the new debug images artifact, and not run the tests. After upload, a separate workflow could then be triggered to download the debug images, and run pytest. However,  I am unsure how to do this. It may require a new token.
https://github.com/actions/download-artifact?tab=readme-ov-file#download-artifacts-from-other-workflow-runs-or-repositories

I may need help to make this change, either with this PR directly or as a separate follow-up PR.

Reply:
If the token is needed, the administrator will take care of it, so please make a suggestion. We will review the they use of token. Alternatively, storing the images in a separate repository may be a good idea to accommodate this.

Could you mention me when you have decided on a policy?

I would love to merge this PR in best way as it is a great.

Reply:
I also think adding extra steps and storing images in a separate repo adds extra complexity and delay... not sure it's worth the trade off. 

As it is, running the rests in the docs workflow with the ~70MB image cache stored in the `pyvista` repo locally only takes 2mins to run: 1 min for pre-processing and 1 min for the actual tests. And running them immediately after the build finishes without having to wait for any uploads means that the current config adds little overhead to the CI testing.

Reply:
Thank you for presenting specific numbers. With that capacity and speed, it is worth storing in the cache in my mind.

Reply:
Update: https://github.com/pyvista/pyvista/pull/5718/commits/d7655505a5d1a2dcfec2f2e1ee66d69448ed9eb2 saves the images as JPG instead of PNG. This reduces the size of the cache from ~70MB to ~20MB with approx 2200 images. 

Compare this to approx 12MB for the plotting/image_cache with 470 images.